content/events/2018/competition.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Competition 2018
+date: 2018-04-21 09:00:00
+end_date: 2018-04-22 17:00:00
+all_day: false
+---
+
+{{% location cube %}}
+
+## Schedule
+The competition will run over 2 days, starting on Saturday morning, and finishing on Sunday afternoon. Exact times to be confirmed.
+
+{{% media-consent%}}
```

Comment:
minor: could we balance the spacing around the name here? `{{% media-consent%}}` -> `{{% media-consent %}}`